.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:10.16.0
+      - image: circleci/node:12
```

Comment:
Any possibility to get onto the latest LTS? v16

Reply:
Also is it possible to break off circleci for github actions.  
It is good to have alternative paths and all, except that circleci time credits expire really quickly.

Reply:
better to have a separate ticket/PR to move the ci/cd pipeline to github action, i'll create one

Reply:
will implement this on https://servicesmadesimpler.govnet.qld.gov.au/jira/browse/QOL-9334